flows/webhook.go

```diff
@@ -192,10 +216,13 @@ func newWebhookCallFromResponse(requestTrace string, response *http.Response, ma
 			return nil, errors.Errorf("webhook response body exceeds %d bytes limit", maxBodyBytes)
 		}
 
+		if len(bodySniffed) > 0 {
+			bodyBytes = append(bodySniffed, bodyBytes...)
+		}
+
 		w.responseTrace += string(bodyBytes)
 	} else {
-		// no body for non-text responses but add it to our Response log so users know why
-		w.responseTrace += "Non-text body, ignoring"
```

Comment:
So instead of faking a response body `"Non-text body, ignoring"`, the event will have `body_ignored=true`. Up to the caller what they want to fake or not.